manifests/integrations/redis.pp

```diff
@@ -37,6 +37,7 @@
   Array $keys                               = [],
   Boolean $warn_on_missing_keys             = true,
   Boolean $command_stats                    = false,
+  Optional[Array] $instances                = undef,
```

Comment:
Not documented. Also, we could mention "ports" is deprecated in favor of this now.

Reply:
I'll document this. This is not deprecated in the integration itself, so I'd rather not confuse the user.... 🤔